src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -119,6 +119,7 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
                 .max="${stateObj.attributes.media_duration}"
                 .value="${stateObj.attributes.media_position}"
                 class="progress"
```

Comment:
Add a classMap that adds a class when it supports `SEEK` and only add the hover height in that class.